src/runtime-rs/crates/shim/src/shim_start.rs

```diff
@@ -105,22 +104,6 @@ impl ShimExecutor {
 
         Ok(child.id())
     }
-
-    fn get_shim_info_from_sandbox(&self, sandbox_id: &str) -> Result<(PathBuf, u32)> {
```

Comment:
One user left at https://github.com/kata-containers/kata-containers/blob/e57e1122c9850816999efdc26ebdf2bc344e64fe/src/runtime-rs/crates/shim/src/shim_start.rs#L197.

`test_get_info_from_sandbox()` should be removed as well.

Reply:
Oh I missed that, good catch, thanks!

Reply:
This should be fixed now.